docs/Using-Pantheon/Accessing-Logs-Using-JSON-RPC.md

```diff
@@ -0,0 +1,164 @@
+# Accessing Logs Using JSON-RPC API Methods
+
+Access logs using JSON-RPC API methods:
+ 
+* [`eth_getFilterChanges`](../Reference/JSON-RPC-API-Methods.md#eth_getfilterchanges)
+* [`eth_getFilterLogs`](../Reference/JSON-RPC-API-Methods.md#eth_getfilterlogs)
+* [`eth_getLogs`](../Reference/JSON-RPC-API-Methods.md#eth_getlogs)
+
+Before using [`eth_getFilterChanges`](../Reference/JSON-RPC-API-Methods.md#eth_getfilterchanges) and 
+[`eth_getFilterLogs`]((../Reference/JSON-RPC-API-Methods.md#eth_getfilterlogs)), create the filter 
+using [`eth_newFilter`](../Reference/JSON-RPC-API-Methods.md#eth_newfilter).
```

Comment:
I'd suggest rephrasing the methods in this order: "Use `eth_newFilter` to create a filter before using `eth_getFilterChanges` and `eth_getFilterLogs`."

Reply:
Done